infoenergia_api/contrib/contracts.py

```diff
@@ -136,6 +136,22 @@ def tertiaryPower(self):
          'P{}'.format(i): int(period['potencia'] * 1000) for i, period in enumerate(period_powers, 1)
         }
 
+    @property
+    def tertiaryPowerHistory(self):
+        test = [
+            {
+                "power": {
+                    str(period[0]).split()[0]: float(period[1]) * 1000
```

Comment:
i think this expresion is a little complex (but very nice :D) what do you think if we encapsulete it in a function named `get_power_history(modcon)`??
the structure will be clearer
```python
{
    "power": get_power_history(modcon),
    "dateStart": make_utc_timestamp(modcon['data_inici']),
    "dateEnd": make_utc_timestamp(modcon['data_final'])
} ....
```